src/main/js/main_html5.js

```diff
@@ -1091,6 +1116,11 @@ require("../../../html5-common/js/utils/environment.js");
       if (firstPlay) {
         this.controller.notify(this.controller.EVENTS.ASSET_DIMENSION, {width: _video.videoWidth, height: _video.videoHeight});
       }
+
+      var availableAudio = this.getAvailableAudio();
+      if (availableAudio) {
```

Comment:
If we change the return type of getAvailableAudio when there is no available audio, this if statement will need to get adjusted.

Reply:
\+